stoobly_agent/app/cli/scaffold/workflow_validate_command.py

```diff
@@ -25,62 +26,90 @@ def __init__(self, app: App, **kwargs):
-    if not gateway_container or (gateway_container.status != 'running'):
-      raise ScaffoldValidateException(f"Container '{gateway_container_name}' not found for service '{CORE_GATEWAY_SERVICE_NAME}'")
 
-    print(f"Validating core component: {CORE_MOCK_UI_SERVICE_NAME}")
-    mock_ui_container_name = self.managed_services_docker_compose.mock_ui_container_name
-    mock_ui_container = self.docker_client.containers.get(mock_ui_container_name)
-    if not mock_ui_container or (mock_ui_container.status != 'running'):
-      raise ScaffoldValidateException(f"Container '{mock_ui_container_name}' not found for service '{CORE_MOCK_UI_SERVICE_NAME}'")
+    container_missing_message = f"{bcolors.FAIL}Container '{gateway_container_name}' not found for service '{CORE_GATEWAY_SERVICE_NAME}'{bcolors.ENDC}"
+    suggestion_message = f"{bcolors.BOLD}Workflow might not be running yet. Try running 'scaffold workflow up <WORKFLOW_NAME>'{bcolors.ENDC}"
+    error_message = f"{container_missing_message}\n\n{suggestion_message}"
 
-  def validate_no_core_components(self):
     try:
-      core_mock_ui_container_name = self.docker_client.containers.get(self.managed_services_docker_compose.mock_ui_container_name)
-      if core_mock_ui_container_name:
-        raise ScaffoldValidateException(f"Stoobly UI container is running when it shouldn't: {core_mock_ui_container_name.name}")
+      gateway_container = self.docker_client.containers.get(gateway_container_name)
+      if not gateway_container or (gateway_container.status != 'running'):
+        raise ScaffoldValidateException(error_message)
     except docker_errors.NotFound:
-      pass
-    
-    print(f"Skipping validating core component: {CORE_GATEWAY_SERVICE_NAME}")
-    print(f"Skipping validating core component: {CORE_MOCK_UI_SERVICE_NAME}")
+      raise ScaffoldValidateException(error_message)
 
-  
-  def validate(self) -> bool:
-    print(f"Validating workflow: {self.workflow_name}")
-    print(f"Validating core components: {CORE_SERVICES}")
+  def validate_mock_ui_service(self):
+    mock_ui_container_name = self.managed_services_docker_compose.mock_ui_container_name
 
+    # The stoobly-ui service does not run in test workflows
     if self.workflow_name == WORKFLOW_TEST_TYPE:
-      # Don't validate the gateway and mock_ui core components in the "test" workflow 
-      self.validate_no_core_components()
-    else:
-      self.validate_core_components()
+      try:
+        mock_ui_container = self.docker_client.containers.get(mock_ui_container_name)
+
+        if mock_ui_container:
+          ui_found_message = f"{bcolors.FAIL}Stoobly UI container is running when it shouldn't: {mock_ui_container_name}{bcolors.ENDC}"
+          suggestion_message = f"{bcolors.BOLD}Run 'docker ps | grep stoobly_ui'. If the UI is running during your test workflow, then report a bug at https://github.com/Stoobly/stoobly-agent/issues'{bcolors.ENDC}"
```

Comment:
This could be because the user did not stop the mock workflow, we should be recommending this first.